NickvisionMoney.Shared/Controllers/TransferDialogController.cs

```diff
@@ -215,4 +218,56 @@ public TransferCheckStatus UpdateTransfer(string destPath, string? destPassword,
         Transfer.ConversionRate = conversionRate;
         return TransferCheckStatus.Valid;
     }
+
+    private static Dictionary<string, string> tmpCacheSoIDontGetRateLimited = new();
```

Comment:
I think this can be called `_ratesCache` or something like this. Add comment if you want to explain why this is needed (although it's pretty obvious to me).